src/lib/exposes.ts

```diff
@@ -542,6 +542,13 @@ export class Climate extends Base {
         this.features.push(schedule);
         return this;
     }
+
+    withFrostProtectionTemperature(min=4.0, max=35.0, step=0.5, access =a.ALL) {
```

Comment:
Please move this out of the climate expose (and just add it to the `exposes: []`) of the device. We don't want to pollute the climate expose.

Reply:
No problem, will move it later this evening.